sdcm/scan_operation_thread.py

```diff
@@ -451,7 +451,7 @@ def execute_query(self, session, cmd: str,
         self.log.debug('Will run command %s', cmd)
         try:
             cmd_result = session.execute(
-                query=cmd, trace=True, timeout=self._session_execution_timeout)
```

Comment:
there isn't any code that depends on it ? i.e. there is no validation ?

Reply:
you rihgt. probably it is. will run test job to be sure

Reply:
@fruch, I've  updated PR
and tested here:
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/artsiom_mishuta/job/longevity-10gb-3h-test/5/